<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLPlusServerLauncherRemoteWidget</class>
 <widget class="qMRMLWidget" name="qMRMLPlusServerLauncherRemoteWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plus Server Launcher Remote</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="HostnameLabel">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Hostname:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="HostnameLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>localhost</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="PortLabel">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="PortLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>18904</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="qMRMLNodeComboBox" name="PlusServerLauncherComboBox">
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLPlusServerLauncherNode</string>
        </stringlist>
       </property>
       <property name="noneEnabled">
        <bool>true</bool>
       </property>
       <property name="renameEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Launcher node:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="qMRMLPlusServerLauncherTableView" name="PlusServerLauncherTable"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLPlusServerLauncherTableView</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLPlusServerLauncherTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qMRMLPlusServerLauncherRemoteWidget.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>qMRMLPlusServerLauncherRemoteWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>PlusServerLauncherTable</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PlusServerLauncherComboBox</sender>
   <signal>currentNodeChanged(vtkMRMLNode*)</signal>
   <receiver>PlusServerLauncherTable</receiver>
   <slot>setLauncherNode(vtkMRMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qMRMLPlusServerLauncherRemoteWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>PlusServerLauncherComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
